src/com/artos/annotation/AfterTestUnit.java

```diff
@@ -40,7 +40,9 @@
 // Make the annotation available at runtime:
 @Retention(RetentionPolicy.RUNTIME)
 // Allow to use only on types:
-@Target(ElementType.METHOD)
+@Target({ ElementType.TYPE, ElementType.METHOD })
 public @interface AfterTestUnit {
+	
+	String[] group() default { "*" };
```

Comment:
@SwapnaSoni  we do not need this addition. User can apply @Group annotation to the method which is annotated using @AfterTestUnit, they should not need to add group array within @AfterTestUnit attribute